bower.json

```diff
@@ -17,14 +17,20 @@
   ],
   "dependencies": {
     "purescript-prelude": "^3.0.0",
-    "purescript-parsing": "^4.0.0",
+    "purescript-parsing": "^4.2.1",
     "purescript-fixed-points": "^4.0.0",
-    "purescript-datetime": "^3.0.0"
+    "purescript-datetime": "git://github.com/safareli/purescript-datetime.git#interval",
```

Comment:
🛑  should be changed after  https://github.com/purescript/purescript-datetime/pull/52 is merged